interchain-token-service/src/primitives.rs

```diff
@@ -1,6 +1,8 @@
+use std::fmt::Display;
+
```

Comment:
I dislike using `primitives.rs` files, it's sort of a catch all, which gets very messy. You could just call this `its_message.rs` or somesuch.

Reply:
Open to other naming suggestions. I don't like `its_message.rs` since it's too specific to the message type, but this file does contain core ITS types (like `TokenId`, `TokenManagerType`) that are relevant independent of the message itself.

Reply:
I don't like catch-all files either, but if it's just TokenId and ItsMessage I think it's fine. It just mustn't grow further than this

Reply:
preserve primitives.rs for now